spot-contracts/contracts/RolloverVault.sol

```diff
@@ -482,14 +460,19 @@ contract RolloverVault is
         whenNotPaused
         returns (uint256 perpAmt, uint256 vaultNoteAmt, TokenAmount[] memory returnedTokens)
     {
-        // Compute perp vault split
         IPerpetualTranche perp_ = perp;
-        (perpAmt, vaultNoteAmt) = feePolicy.computeDRNeutralSplit(
-            perpAmtAvailable,
-            vaultNoteAmtAvailable,
-            perp_.totalSupply(),
-            totalSupply()
-        );
+
+        // Compute perp vault split
+        {
```

Comment:
There are a few places where you added extra namespaces/nesting. It's a pattern that's useful in C++ for RAII, but don't think that's relevant here. Is there another reason? If it's for documentation, I'd rather do that in docstrings rather than something that could potentially impact the compiled output.

(and other locations)

Reply:
The compiler does limit the number of variables in a block to 16, If I remember correctly. So using blocks, helps keep the function namespace cleaner (ie ones declared in the block wont propagate up). I've used them from time to time specifically for that reason. But here, we are nowhere close to 16 variables.